.github/workflows/build.yml

```
@@ -169,12 +169,50 @@ jobs:
         uses: mxschmitt/action-tmate@v3
         if: env.RUN_TMATE
   test:
+    name: >-
+      test (${{ matrix.scenario }}) -
+      ${{ matrix.platform }}-${{ matrix.architecture }}
     needs:
       - diagnostics
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
+        architecture:
+          - amd64
+          # CobaltStrike does not support ARM64.
+          # https://github.com/cisagov/ansible-role-cobalt-strike/pull/85#issuecomment-2171882459
+          # - arm64
+        platform:
+          # # Only Debian-based platforms are currently supported.
+          # - amazonlinux2023-systemd
+          # The latest version of CobaltStrike requires glibc 2.29,
+          # which is unavailable for Debian Buster.  When one attempts
+          # to start the teamserver on Buster one gets the following
+          # output:
+          #
+          # root@debian10:/# cd /opt/cobaltstrike/
+          # root@debian10:/opt/cobaltstrike# ./teamserver
+          #
+          # [*] Checking TeamServerImage for local update
+          #
+          # [*] Verifying MD5 Message Digest for TeamServerImage
+          # TeamServerImage: OK
+          #
+          # [*] Will use existing X509 certificate and keystore (for SSL)
+          #
+          # [*] Starting teamserver
+          # ./TeamServerImage: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by ./TeamServerImage)
+          # - debian10-systemd
+          - debian11-systemd
+          - debian12-systemd
+          - debian13-systemd
+          # - fedora39-systemd
```

Just as a reminder for why these are disabled.

```suggestion
          # # Only Debian-based platforms are currently supported.
          # - fedora39-systemd
```